index.bs

```diff
@@ -1445,6 +1454,16 @@ When this method is invoked, the user agent MUST execute the following algorithm
                 on |authenticator| and [=set/remove=] it from |issuedRequests|.
 
             1.  Return |constructAssertionAlg| and terminate this algorithm.
+
+        :   If the [=client=] concludes that it is impossible for any eligible [=authenticator=] to appear,
+        ::  The client MAY prompt the user for [=user consent|consent=] to create a new credential. If the user does [=user
```

Comment:
We are not asking for consent to create a new cred here, rather to generate an assertion ie to authenticate, yes?

Reply:
should "user does" be "user does not" ?

Reply:
Good catch, thanks! See https://github.com/w3c/webauthn/pull/962#discussion_r197862736 for my answer to the second question.

Reply:
Heh, I had swapped not only the Note:s between algorithms, but in fact the whole algorithm steps...